src/plugins/audiobridge-plugin.js

```diff
@@ -787,6 +798,52 @@ class AudioBridgeHandle extends Handle {
     throw (error);
   }
 
+  /**
+   * Mute the given room for every participant.
+   * 
+   * @param {object} params
+   * @param {number|string} params.room - The involved room
+   * @param {string} [params.secret] - The optional secret needed to manage the room
+   * @returns {Promise<module:audiobridge-plugin~AUDIOBRIDGE_EVENT_MUTE_ROOM_RESPONSE>}
+   */
+  async muteRoom({ room, secret }) {
+    const body = {
+      request: REQUEST_MUTE_ROOM,
+      room,
+    };
+    if (typeof secret === 'string') body.secret = secret;
+
+    const response = await this.message(body);
+    const { event, data: evtdata } = response._janode || {};
+    if (event === PLUGIN_EVENT.SUCCESS)
+      return evtdata;
```

Comment:
according to janus docs, room is not part of the success response.
Could you please copy the `room` id from the request body? e.g.

```
if (event === PLUGIN_EVENT.SUCCESS) {
      evtdata.room = body.room;
      return evtdata;
}
```